core/src/main/java/org/mule/runtime/core/internal/policy/CompositeSourcePolicy.java

```diff
@@ -239,15 +228,14 @@ private MessagingException mapErrorInInnerExecution(MessagingException messaging
    * @return a {@link Publisher} that emits {@link SourcePolicySuccessResult} which contains the response parameters and the
    *         result event of the execution or a {@link SourcePolicyFailureResult} which contains the failure response parameters
    *         and the {@link MessagingException} thrown by the policy chain execution when processing completes.
-   * @throws Exception if there was an unexpected failure thrown by executing the chain.
    */
   @Override
   public Publisher<Either<SourcePolicyFailureResult, SourcePolicySuccessResult>> process(CoreEvent sourceEvent,
                                                                                          MessageSourceResponseParametersProcessor respParamProcessor) {
     return commonPolicy.process(sourceEvent, respParamProcessor);
   }
 
-  protected Supplier<Map<String, Object>> resolveErrorResponseParameters(MessagingException e) {
+  private Supplier<Map<String, Object>> resolveErrorResponseParameters(MessagingException e) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this method into "SourceWithPoliciesFluxObjectFactory". ([squid:S3398](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3398))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7760&issues=AWnGD5CXotIXnmsUaS7l&open=AWnGD5CXotIXnmsUaS7l)

[ISSUE_KEY]: AWnGD5CXotIXnmsUaS7l